### ISSUE TITLE
Build instructions are incorrect

### ISSUE THREAD

**Issue by alejandro-colomar** [include-what-you-use/include-what-you-use#962]
Hi, there are a few problems in the build instructions in the readme:

- The cmake command line is incorrect.  Instead of `../include-what-you-use`, it should say simply `..`.
- The explanation about branches for clang versions is very confusing.  It seems like you have many versions of the master branch with different configurations for different clang versions, but it's simply that you map old iwyu versions to old clang versions, but the latest iwyu version is only prepared to build with the latest clang version (and if you're lucky, a few more).  I'd reword that.

**Comment by kimgr** [comment]
I have some doc changes to the README lined up in #1032, so I'm going to close this.
